src/adapter-types/src/dyncfgs.rs

```diff
@@ -84,6 +84,13 @@ pub const DEFAULT_SINK_PARTITION_STRATEGY: Config<&str> = Config::new(
     "The default sink partitioning strategy for an environment. It defaults to 'v0'.",
 );
 
+/// Whether to log the status of pgwire connections.
+pub const LOG_PGWIRE_CONNECTION_STATUS: Config<bool> = Config::new(
+    "log_pgwire_connection_status",
+    false,
+    "Whether to log the status of pgwire connections.",
+);
+
```

Comment:
Suggest using tracing's builtin filtering for this instead of adding a one off config. See later comments!

Reply:
I've removed this.